services/QuillLMS/app/models/concerns/teacher.rb

```diff
@@ -322,6 +322,10 @@ def classroom_minis_info
     info
   end
 
+  def teaches_eighth_through_twelfth?
+    return classrooms_i_teach.map(&:grade).compact.any? { |grade| grade.to_i <= 12 && grade.to_i >= 8 }
```

Comment:
[between?](https://ruby-doc.org/core-2.4.0/Comparable.html#method-i-between-3F) might be more readable here:

`classrooms_i_teach.map(&:grade).compact.any? { |grade| grade.to_i.between?(8,12) }`

Reply:
"Twelfth" is a word that will always look spelled wrong to me even though it is in fact correct.

Reply:
@emilia-friedberg I know!!! I had to google "twelfth" to make sure.